R/deprecated.R

```diff
@@ -1,44 +1,44 @@
+
 #' Print message for deprecated functions in Shiny
 #'
 #' To disable these messages, use `options(shiny.deprecation.messages=FALSE)`.
 #'
-#' @param new Name of replacement function.
-#' @param msg Message to print. If used, this will override the default message.
-#' @param old Name of deprecated function.
-#' @param version The last version of Shiny before the item was deprecated.
+#' @param version Shiny version when the function was deprecated
+#' @param what Function with possible arguments
+#' @param with Possible function with arguments that should be used instead
+#' @param details Additional information to be added after a new line to the displayed message
 #' @keywords internal
```

Comment:
I suppose we should mention the breaking changes in NEWS (and, more importantly, add NEWS items?)

Reply:
`shinyDeprecated` was never exported. It just had an Rd file.

Reply:
Added in 4091b39

Reply:
Thanks!